app/assets/javascripts/components/overview/details.jsx

```diff
@@ -314,62 +315,97 @@ const Details = createReactClass({
       );
     }
 
-    return (
-      <div className="module course-details">
-        <div className="section-header">
-          <h3>{I18n.t('application.details')}</h3>
-          {this.props.controls()}
+    if (!this.props.editable) {
+      return (
```

Comment:
Could this be done without duplicating all the content, by setting the shared content to a constant first, and then just switching on whether to wrap that content in `Modal`?